server/http_status.go

```diff
@@ -85,9 +87,38 @@ func (s *Server) startHTTPServer() {
 	serverMux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
 	serverMux.HandleFunc("/debug/pprof/trace", pprof.Trace)
 
+	var (
+		err            error
+		httpRouterPage bytes.Buffer
+		pathTemplate   string
+	)
+	httpRouterPage.WriteString("<html><head><title>TiDB Status and Metrics Report</title></head><body><h1>TiDB Status and Metrics Report</h1><table>")
+	err = router.Walk(func(route *mux.Route, router *mux.Router, ancestors []*mux.Route) error {
+		pathTemplate, err = route.GetPathTemplate()
+		if err != nil {
+			log.Error("Get http router path err", err)
+		}
+		name := route.GetName() //if not set name attribute,GetName return ""
```

Comment:
s/`if not set name attribute,GetName return ""`/`If the name attribute is not set, GetName returns "".`